src/styles.css

```diff
@@ -437,7 +437,13 @@ h3 {
   filter: blur(4px);
 }
 
-#clickedRecipe {
+#settingsPanel {
+  flex-direction: column;
+  justify-content: normal;
+  align-items: flex-start;
+}
```

Comment:
is it possible to make sure the chart does not bleed out of the bottom of the modal?
<img width="1228" alt="Screen Shot 2023-06-03 at 9 18 24 PM" src="https://github.com/lauraguerra1/whats-cookin/assets/121131581/07c6f943-abc6-4f23-bf62-60238e4a2694">

Reply:
Great call!! Even though I had `responsive: true` set in the chart options, turns out I also needed the parent element to have some additional styling and another option of `maintainAspectRatio: false` in order for it to look nicely responsive.

Reply:
thats awesome, nicely done!